### DISCUSSION TITLE
[Enhancement] [date-picker] 日期选择器组件可以只显示选择器的面板不需要选择框

### DISCUSSION THREAD

**Discussion by mingmin74** [element-plus/element-plus#18298] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

日期选择器组件可以只显示选择器的日历面板而不需要选择框

### Additional comments

_(empty)_

![image](https://github.com/user-attachments/assets/3f72ea29-8cf8-468a-8ece-e8b1816ace40)

**Comment by Dsaquel** [comment]
Released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html
